mscore/pm.cpp

```diff
@@ -63,22 +66,23 @@ PortMidiDriver::~PortMidiDriver()
 bool PortMidiDriver::init()
       {
       inputId = getDeviceIn(preferences.portMidiInput);
-      if(inputId == -1)
-        inputId  = Pm_GetDefaultInputDeviceID();
-      outputId = Pm_GetDefaultOutputDeviceID();
+      if (inputId == -1)
+            inputId  = Pm_GetDefaultInputDeviceID();
 
       if (inputId == pmNoDevice)
             return false;
 
-      static const int INPUT_BUFFER_SIZE = 100;
```

Comment:
I've moved this to preferences (not the gui, but will be in the actual preferences file).